mock_server/api.py

```diff
@@ -1,11 +1,29 @@
-def bookings_options_get(message):
+import json
+import flask
+
+
+# TODO - how to cope with optional parameters?
+# TODO - from as Parameter name is not a good choice as the mapping here leads to an invalid code syntax ? 
+# TODO -    how to map parameters in API to handler parameters?
+# - 
+# changed from to fromP because from is a reserved word in Python
+def bookings_options_get(_from, to, startTime):
```

Comment:
@ChristianMaehler, do this instead:

`def bookings_options_get(**options):`

Then we can keep it named as "from". In your implementation you then reference it as `options["from"]`. I just tested it and it works.

Also handles optional parameters better and you don't need to change the method signature whenever there's anew parameter.

Reply:
Very good suggestion, thank you very much! I committed that change ...